README.md

```diff
@@ -17,7 +17,9 @@ features for JSON Pointers as well.
 
 ## Documentation
 
-The documentation for this project can be found here: https://github.com/whitlockjc/json-refs/blob/master/docs/README.md
+The documentation for this project can be found here: https://github.com/whitlockjc/json-refs/blob/master/docs/README.md .
+
+The API can be found at https://github.com/whitlockjc/json-refs/blob/master/docs/API.md .
```

Comment:
I've surrounded the URLs with brackets and removed the spaces preceding the periods.

My niggling here might have to do with my having been an English teacher before. :)